irohad/consensus/yac/transport/yac_pb_converters.hpp

```diff
@@ -35,10 +37,22 @@ namespace iroha {
           hash->set_proposal(vote.hash.proposal_hash);
 
           auto block_signature = hash->mutable_block_signature();
-          block_signature->set_signature(
-              vote.hash.block_signature.signature.to_string());
-          block_signature->set_pubkey(
-              vote.hash.block_signature.pubkey.to_string());
+
+          //Will fix it in the next PR, very soon, don't worry
+          if (vote.hash.block_signature == nullptr) {
+            auto sig = shared_model::proto::SignatureBuilder()
```

Comment:
There is a `builders/common_objects/signature_builder.hpp` which returns interface signature. Please add this builder to `default_builders.hpp` as it is done for other builders.